monai/transforms/smooth_field/dictionary.py

```diff
@@ -0,0 +1,148 @@
+
+import numpy as np
+
+from monai.config import KeysCollection
+from monai.transforms.smooth_field.array import RandSmoothFieldAdjustContrast, RandSmoothFieldAdjustIntensity
+from monai.transforms.transform import MapTransform, Randomizable, RandomizableTransform, Transform
+from monai.utils import InterpolateMode
+
+__all__ = ["RandSmoothFieldAdjustContrastd", "RandSmoothFieldAdjustIntensityd"]
+
+
+class RandSmoothFieldAdjustContrastd(RandomizableTransform, MapTransform):
+    """
+    Dictionary version of RandSmoothFieldAdjustContrast. The field is randomized once per invocation by default so the
+    same field is applied to every selected key.
+
+    Args:
+        keys: key names to apply the augment to
+        spatial_size: size of input arrays, all arrays stated in `keys` must have same dimensions
+        rand_size: size of the randomized field to start from
+        padder: optional transform to add padding to the randomized field
+        mode: interpolation mode to use when upsampling
+        align_corners: if True align the corners when upsampling field
+        prob: probability transform is applied
+        gamma: (min, max) range for exponential field
+        apply_same_field: if True, apply the same field to each key, otherwise randomize individually
+    """
+
+    def __init__(
+        self,
+        keys: KeysCollection,
+        spatial_size: Union[Sequence[int], int],
+        rand_size: Union[Sequence[int], int],
+        padder: Optional[Transform] = None,
+        mode: Union[InterpolateMode, str] = InterpolateMode.AREA,
+        align_corners: Optional[bool] = None,
+        prob: float = 0.1,
+        gamma: Union[Sequence[float], float] = (0.5, 4.5),
+        apply_same_field: bool = True,
+    ):
+        RandomizableTransform.__init__(self, prob)
+        MapTransform.__init__(self, keys)
+
+        self.trans = RandSmoothFieldAdjustContrast(spatial_size, rand_size, padder, mode, align_corners, 1.0, gamma)
+        self.apply_same_field = apply_same_field
+
+    def set_random_state(
+        self, seed: Optional[int] = None, state: Optional[np.random.RandomState] = None
+    ) -> "RandSmoothFieldAdjustContrastd":
+        super().set_random_state(seed, state)
+        self.trans.set_random_state(seed, state)
+        return self
+
+    def randomize(self, data: Optional[Any] = None) -> None:
+        super().randomize(None)
+        self.trans.randomize()
+
+    def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
+        self.randomize()
+
+        if not self._do_transform:
+            return data
+
+        d = dict(data)
+
+        for key in self.key_iterator(d):
+            if not self.apply_same_field:
+                self.randomize()  # new field for every key
+
+            d[key] = self.trans(d[key], False)
+
+        return d
+
+
+class RandSmoothFieldAdjustIntensityd(RandomizableTransform, MapTransform):
```

Comment:
Could you set the transform backends (eg https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/croppad/array.py#L89)? I suspect it will be trivial to make them compatible with both np and torch inputs.